src/AzureOpenAIProxy.Management/Services/ModelService.cs

```diff
@@ -1,25 +1,62 @@
+using System.Data;
 using AzureOpenAIProxy.Management.Components.ModelManagement;
 using AzureOpenAIProxy.Management.Database;
 using Microsoft.EntityFrameworkCore;
+using Npgsql;
+using NpgsqlTypes;
 
 namespace AzureOpenAIProxy.Management.Services;
 
-public class ModelService(IAuthService authService, AoaiProxyContext db) : IModelService
+public class ModelService(IAuthService authService, AoaiProxyContext db, IConfiguration configuration) : IModelService
 {
+
+    private const string PostgresEncryptionKey = "PostgresEncryptionKey";
+    private readonly NpgsqlConnection connection = (NpgsqlConnection)db.Database.GetDbConnection();
```

Comment:
Does this need to be a `NpgsqlConnection` or is `DbConnection` fine? Elsewhere we just use `DbConnection`